src/lib/Link.js

```diff
@@ -21,7 +21,7 @@ const Button = (props) => (
 );
 
 Button.propTypes = {
-  children: PropTypes.string,
+  children: PropTypes.any,
```

Comment:
Oh, could you by the way refactor `Button` -> `Link` ?

Reply:
It was done in other PR, I'll rebase it